### PR TITLE
Some light mode changes

### PR DESCRIPTION
**Pull request by GrahamSH-LLK**
fixes up some minor stuffs

### CODE REVIEW

File: webpages/settings/index.js [hunk 1]
```diff
@@ -787,8 +787,7 @@ const vue = (window.vue = new Vue({
     },
     setTheme(mode) {
       chrome.storage.sync.get(["globalTheme"], function (r) {
-        let rr = true; //true = light, false = dark
-        rr = mode;
+        let rr = mode;
```
Comment (TheColaber) [inline webpages/settings/index.js:790]: why did you remove this logic?

Comment (ErrorGamer2000) [reply webpages/settings/index.js:790]: It seems to me that it will be changed no matter what, and it is just unnecessary.

Comment (GrahamSH-LLK) [reply webpages/settings/index.js:790]: Yeah, if you look at the old logic it sets it to true then proceeded to change it to mode

Comment (TheColaber) [reply webpages/settings/index.js]: im now realizing that the logic is broken

File: webpages/settings/index.js [hunk 2]
```diff
@@ -787,8 +787,7 @@ const vue = (window.vue = new Vue({
     },
     setTheme(mode) {
       chrome.storage.sync.get(["globalTheme"], function (r) {
-        let rr = true; //true = light, false = dark
-        rr = mode;
+        let rr = mode || true;
```
Comment (TheColaber) [inline webpages/settings/index.js:790]: This makes it true no matter what

Comment (ErrorGamer2000) [reply webpages/settings/index.js:790]: Why?

Comment (ErrorGamer2000) [reply webpages/settings/index.js:790]: maybe try `mode??true`?

Comment (TheColaber) [reply webpages/settings/index.js:790]: that will work

File: webpages/settings/index.js [hunk 3]
```diff
@@ -787,8 +787,7 @@ const vue = (window.vue = new Vue({
     },
     setTheme(mode) {
       chrome.storage.sync.get(["globalTheme"], function (r) {
-        let rr = true; //true = light, false = dark
-        rr = mode;
+        let rr = mode ?? true;
```
Comment (WorldLanguages) [inline webpages/settings/index.js:790]: We can use `?.` but not `??`, that's only Chrome 85+

Comment (WorldLanguages) [inline webpages/settings/index.js:790]: lol `??` is chrome 80+, we almost cant use it

Comment (ErrorGamer2000) [reply webpages/settings/index.js:790]: Oof

**Comment by WorldLanguages** [comment]
Regression: 
image

**Comment by mxmou** [comment]
@WorldLanguages That could be fixed by using the `addon-description-full` class.

Did this PR intentionally change the layout of the modal? It didn't use different classes for no reason.

**Comment by GrahamSH-LLK** [comment]
> @WorldLanguages That could be fixed by using the `addon-description-full` class.
> 
> Did this PR intentionally change the layout of the modal? `settings-*` classes weren't used for no reason.

Sorry, I made an assumption that they were the same because they were when I designed it 😳

**Comment by mxmou** [comment]
> Sorry, I made an assumption that they were the same because they were when I designed it 😳

I changed them quite a long time ago  (#1418), but they're still mostly the same. I think using the same classes is a good idea, but layout-related parts of the previous modal-specific styles should be kept.
